src/order/filter/countable_Inter.lean

```diff
@@ -189,3 +189,70 @@ begin
   refine ⟨λ S hSc hS, ⟨_, _⟩⟩; refine (countable_sInter_mem hSc).2 (λ s hs, _),
   exacts [(hS s hs).1, (hS s hs).2]
 end
+
+namespace filter
+
+variable (g : set (set α))
+
+/-- `countable_generate_sets g` is the (sets of the)
+greatest `countable_Inter_filter` containing `g`.-/
```

Comment:
Mathlib docs autolink def/lemma names in docstrings but only if you use full names with namespaces.
```suggestion
/-- `filter.countable_generate_sets g` is the (sets of the)
greatest `countable_Inter_filter` containing `g`.-/
```